hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -108,7 +108,7 @@ public final class OzoneManagerDoubleBuffer {
   private final boolean isTracingEnabled;
   private final Semaphore unFlushedTransactions;
   private final FlushNotifier flushNotifier;
-  private final String threadPrefix;
+  private String threadPrefix = "";
```

Comment:
This should be left as it was, and we also need to keep the assignment in the constructor.  Defauilt value of `""` should be set on the corresponding variable in the builder.

```suggestion
  private final String threadPrefix;
```

Reply:
Done! Please take a look if I got it right